<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tradeflowwidget</class>
 <widget class="QWidget" name="tradeflowwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tradeflowwidget</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource/shopicon.qrc">
    <normaloff>:/flow.png</normaloff>:/flow.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>日期选择</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit_before"/>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit_after"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_search">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_chart">
        <property name="text">
         <string>生成折线图</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_back">
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>销售统计</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>总成本：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_tcost">
           <property name="text">
            <string>0.00</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>总销售额：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_tsell">
           <property name="text">
            <string>0.00</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>总销利润：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_tprofit">
           <property name="text">
            <string>0.00</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>日均成本：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_acost">
           <property name="text">
            <string>0.00</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>日均销售额：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_asell">
           <property name="text">
            <string>0.00</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>日均利润：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_aprofit">
           <property name="text">
            <string>0.00</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource/shopicon.qrc"/>
 </resources>
 <connections/>
</ui>
